### ISSUE TITLE
Alphanumeric lists not possible in LaTeX and EPUB

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1431]
I discovered that it is possible to make alphanumeric lists in HTML like this:

```
#!rst

I. Schnittblumen

   a. Rosen
   b. Nelken
   c. Tulpen

A. Topfblumen

   i. Petunien
   ii. Geranien
   iii. Pelargonien
```

Unfortunately the output in LaTeX and EPUB is not what I expected. It is rendered more or less like a normal enumerated list. This can be an issue of EPUB and LaTeX. 

Maybe this should be mentioned in the documentation, if it is not possible have alphanumeric lists in LaTeX and EPUB in the near future. As alphanumeric lists are not mentioned at all in the documentation I marked this a minor issue.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1431
- Originally reported by: Jan Ulrich Hasecke
- Originally created at: 2014-03-25T09:23:36.489

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-25 08:25:03+00:00_

Hmm, that it works in html and not epub is strange. I'll have a look at the stylesheets.

LaTeX is independent of those two, so I can believe nobody implemented that.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-25 08:29:47+00:00_

OK, epub is fixed, latex needs a little more work.

**Comment by tk0miya** [comment]
Fixed by #5001!
